home/forms/domain_model.py

```diff
@@ -12,109 +16,22 @@ class DomainModel:
     Store information about domains and subdomains
     """
 
-    def __init__(self):
+    def __init__(self, search_facet_fetcher: SearchFacetFetcher):
```

Comment:
TODO:

- This class should probably live in models, not services
- Allow this to be instantiated with values instead of depending on the service
- Change calling code to use the service instead of instantiating both this and the service